.travis.yml

```diff
@@ -1,3 +1,4 @@
+dist: trusty
```

Comment:
We are having a conection refuse using rsync from server to update avr libc in precise distribution, thats is the only one reason.